src/client/js/partials/notFound.js

```diff
@@ -0,0 +1,19 @@
+if (window.history.length > 1) {
```

Comment:
The license header is missing.

```suggestion
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 if (window.history.length > 1) {
```

Reply:
Fixed in 2f5a84c9a417b67709f2dc007c429f1926b4aac3

BUT it's a waste of your and my time to manually be checking this in reviews, so I also submitted https://github.com/mozilla/blurts-server/pull/2836 as well.

Reply:
Awesome — ty!